external/fv3fit/fv3fit/emulation/data/transforms.py

```diff
@@ -27,31 +27,31 @@ def _register_derived_variables(mapping: DerivedMapping, timestep):
     # the data or configuration.
     @mapping.register(
         "tendency_of_air_temperature_due_to_microphysics",
-        required_inputs=["air_temperature_input", "air_temperature_output"],
+        required_inputs=["air_temperature_input", "air_temperature_after_precpd"],
```

Comment:
These are very brittle hardcodes.  Should probably switch to `Field` definition used by the conservative model and we can build up a derived mapping necessary for sample processing based on that.